server/athenian/api/precompute/accounts.py

```diff
@@ -57,8 +60,18 @@ async def main(context: PrecomputeContext,
     log = context.log
 
     for reposet in tqdm(reposets):
+        duration_tracker = _DurationTracker(args.prometheus_pushgateway, context.log)
+        try:
+            meta_ids = await get_metadata_account_ids(reposet.owner_id, context.sdb, context.cache)
```

Comment:
Not good, we can hang in the DB call here and fuck up the supervisor. Let's discuss on the sync today.

Reply:
changed